Paul-Taiwo/automart
# #166738789 Setup a PostgreSQL database

#### What does this PR do?
Setup a PostgreSQL database

#### Description of Task to be completed?
Setup database to store data instead of the data structures 

#### How should this be manually tested?
- Make sure you have `node.js` installed and a database. 
   Check [here](https://www.guru99.com/postgresql-create-database.htmll) for how to create a database using PostgreSQL
- Clone or fork the repository
- CD into `automart`
- Checkout to `develop` branch
- Run `npm install` to install dependencies
- Run `npm db:migrate` to create tables in the database

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#166738789](https://www.pivotaltracker.com/story/show/166738789)
